CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 11.33
+* Move to using OkHTTP over Dispatch
+
```

Comment:
We need to keep `model` and `scala client` in sync, so we should release a model change before releasing this.

Reply:
If anyone has in mind a versioning version that is both simple to understand for consumer and allow to release easily without such constraint, I will be interested to read you!

Reply:
@mchv move the scala client and the model into two projects within the same repo?

Reply:
I like Regis' suggestion!